openfl/utilities/fedcurv/aggregation_function.py

```diff
@@ -0,0 +1,27 @@
+"""FedCurv Aggregation function module."""
```

Comment:
Should we replace this module to https://github.com/intel/openfl/tree/develop/openfl/component/aggregation_functions and rename it to `fedcurv_aggregation.py` ?